doc/source/remarks.rst

```diff
@@ -33,14 +33,3 @@ Dirty Terminals
 
 If your terminal gets polluted by PyBullet complaining about incomplete URDF descriptions, you need to first fix your
 URDF files by inserting the missing tags and second delete the `resources/cached` folder.
-
-Missing pr2_arm_kinematics
-==========================
```

Comment:
Why did you remove this part?

Reply:
The arm kinematics error was caused by a missing dependency to `moveit_kinematics`. As I added that dependency in ef50689f64ff05106015a3375b037c4037c0b1c4, the error should be gone now.

Reply:
Yeah but that error occurred for some people randomly after some time of usage, so I would like to keep this for documentation at least in the troubleshooting section.

Reply:
As stated in the removed doc, the package `moveit_kinematics` was auto-removed by apt after some time, which happened because the dependency to `moveit_kinematics` was not declared. As it is now, the package will not be removed accidentally anymore. Trust me ;-)
If you insist, I will revert that part.

Reply:
Ok, then I will trust you on that :)